components/blocks/partnerform.tsx

```diff
@@ -1,312 +1,61 @@
-
-const options = [
-    "Booth",
-    "Speaker Opportunities",
-    "Workshops",
-    "Startup Pitching Sessions",
-    "founded. Magazine",
-    "Promotional Opportunities",
-    "More Partnership Opportunities",
-];
+import { useEffect } from "react";
+import type { Template } from "tinacms";
 
 export const PartnerForm = ({ data }) => {
-    const [err, setErr] = useState(false);
-    const [success, setSuccess] = useState(false);
-    const [showSuccess, setShowSuccess] = useState(false);
-    const [loading, setLoading] = useState(false);
-    const [interests, setInterests] = useState([])
-
-    const close = () => {
-        setShowSuccess(false)
-        setErr(false);
-    }
-
-    // Handles the submit event on form submit.
-    const handleSubmit = async (event: any) => {
-        // Stop the form from submitting and refreshing the page.
-        event.preventDefault()
-        setLoading(true);
-
-        const response = await fetch('/api/partner', {
-            method: 'post',
-            headers: {
-                'Content-Type': 'application/json',
-            },
-            body: JSON.stringify({
-                firstname: event.target.firstname.value,
-                lastname: event.target.lastname.value,
-                company: event.target.company.value,
-                interests: interests.join(', '),
-                email: event.target.email.value,
-                budget: event.target.budget.value,
-            }),
-        })
-
-        const { error } = await response.json()
-        setLoading(false);
-        if (error) {
-            setSuccess(false);
-            setErr(true);
-        } else {
-            setErr(false);
-            setSuccess(true);
-            setShowSuccess(true);
-        }
-
-        // remove error messages after 20 seconds
-        setTimeout(() => {
-            close()
-        }, 20 * 1000);
-    }
+    useEffect(() => {
+
+        const script = document.createElement('script');
+        script.src = "//js.hsforms.net/forms/embed/v2.js"
+        document.body.appendChild(script);
+
+        script.addEventListener('load', () => {
+            // @ts-ignore
+            if (window.hbspt) {
+                // @ts-ignore
+                window.hbspt.forms.create({
+                    region: 'eu1',
+                    portalId: '143702882',
+                    formId: 'b0a92bc7-e7df-46e3-a7d1-0a55ccc57621',
+                    target: '#hubspotForm'
+                })
+
+                setTimeout(() => {
+                    const iframe = (document.getElementById('hubspotForm').children[0] as any).contentWindow.document
+                    iframe
+                        .querySelectorAll('style,link[rel="stylesheet"]')
+                        .forEach((item: any) => item.remove())
+                    const style = document.createElement('style')
+                    style.innerHTML = `
+label span {
+    color: red !important;
+}
```

Comment:
Check if typescript produces a sensible name for the generated css file. If so, the css could be added to [`styles.css`](https://github.com/Startup-Nights/website/blob/main/styles.css) and then injected as a link instead of manually writing it. But not sure if it's worth the effort.